doc/provides.rst

```diff
@@ -57,6 +57,8 @@ Provide Categories
 ``admin_module``
     Admin module classes. Practically all of the functionality in the admin is built
     via admin modules.
+``discount_module``
```

Comment:
This should be `price_modifier_module` to be symmetric with `order_source_modifier_module`, then.